docs/source/user/interface.md

```diff
@@ -1,92 +1,86 @@
   code
-- Tabs: a list of the open documents and activities in the Dock Panel
-- Settings: common settings and an advanced Settings Editor
-- Help: a list of JupyterLab and Kernel related help links
+- Tabs: a list of the open documents and activities in the dock panel
+- Settings: common settings and an advanced settings editor
+- Help: a list of JupyterLab and kernel help links
 
-JupyterLab extensions can also create new top-level menus in the Menu Bar.
+JupyterLab extensions can also create new top-level menus in the menu bar.
 
-## Left Panel
+## Left Sidebar
 
-The Left Panel contains a number of commonly used panels, such as the File
-Browser, Running List, Command Palette and Tabs List:
+The left sidebar contains a number of commonly-used tabs, such as a file browser, a list of running kernels and terminals, the command palette, and a list of tabs in the main area:
 
 [screenshot]
 
-The Left Panel can be collapsed or expanded by clicking on the active tab:
+The left sidebar can be collapsed or expanded by clicking on the active sidebar tab:
 
 [animation]
 
-JupyterLab extensions can add additional panels to the Left Panel.
+JupyterLab extensions can add additional panels to the left sidebar.
 
-## Dock Panel
+## Main area
 
-The Dock Panel is the main work area in JupyterLab and allows you to arrange
-documents (notebooks, text files, etc.) and other activities (terminals, code
-consoles, etc.) into panels and tabs that can be resized or subdivided:
+The main work area in JupyterLab allows you to arrange documents (notebooks,
+text files, etc.) and other activities (terminals, code consoles, etc.) into
+panels of tabs that can be resized or subdivided:
 
 [animation]
 
-The Dock Panel has a single active document or activity that receives keyboard
-focus and whose tab is marked with a colored top border (blue by default):
+The main area has a single current activity. The tab for this activity is
+marked with a colored top border (blue by default).
 
 [screenshot]
 
-## Tabs and single-document view
+## Tabs and Single Document Mode
 
-The Tabs List gives you access to the open documents and activities in the Dock
-Panel:
+The Tabs panel in the left sidebar lists the open documents or activities in the
+main area:
 
 [screenshot]
 
-The same information is also available in the Tab Menu:
+The same information is also available in the Tabs Menu:
 
 [screenshot]
 
-It is often useful to focus your work on a single document, while not closing
-other documents. Single Document Mode toggles the view of the Dock Panel to show
-only a single document at a time:
+It is often useful to focus on a single document or activity without closing
+other tabs in the main area. Single Document Mode toggles the view of the main
+area to show only a single tab at a time:
 
 [animation]
 
-When you leave Single Document Mode, the original layout of the Dock Panel is
+When you leave Single Document Mode, the original layout of the main area is
```

Comment:
Are we not calling it the "Dock Panel"?

Reply:
This is part of a bigger conversation I mentioned in my comments, I'd like us to make a decision with @ellisonbg as well since I think his conventions were intentional.